tests/dummy/app/models/appointment-slot.js

```diff
@@ -22,7 +22,7 @@ export default DS.Model.extend({
     }
   }),
 
-  slotPickerStartMoment: computed('convertToUKTimezone', 'startTime', function () {
+  _startMoment: computed('convertToUKTimezone', 'startTime', function () {
```

Comment:
can you add a todo so that the timezone can be a parameter if its intended that this module to be open sourced then probably shouldnt be forced to uk timezone.

Also, do we want _startMoment to be private? You could argue that _startMoment is more useful than startTime?

Reply:
@omersaleem I can, but then the moment timezone library itself is a big library to download, and I would say it's more of a data concern which shouldn't be part of this addon?? (please note I don't even have ember-data has a dependency for this addon).

Reply:
As discussed, I will remove all mentions of .tz from the dummy app - because it's a data concern. And make slotPickerTime work for either a time or a moment.